style.css

```diff
@@ -0,0 +1,165 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: "Montserrat", sans-serif;
+}
+
+.header {
+  width: 100%;
+  height: 200px;
+  background-color: #88c8bc;
+  display: flex;
+  justify-content: center;
+  font-size: xx-large;
+}
+.logo {
+  padding-right: 200px;
+}
+.header .nav-bar {
+  /* border: 2px black solid; */
+  position: relative;
+  top: 50px;
+  width: 70%;
+  display: flex;
+  /* justify-content: space-around; */
+}
+.header .nav {
+  /* border: 2px blue solid; */
+  width: 40%;
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+}
+.search {
+  height: 25px;
+  border-radius: 10px;
+  padding-right: 50px;
+}
+.fa-shopping-cart {
+  padding-right: 300px;
+  padding-left: 50px;
+}
+.fa-search {
+  /* padding-right: 100px; */
+  background-color: #3f9b8a;
+  height: 35px;
+  width: 35px;
+  border-radius: 20px;
+}
+.filter {
+  position: absolute;
+  top: 80px;
+  font-size: large;
+  display: flex;
+  width: 8%;
+  justify-content: space-between;
+  padding-left: 1100px;
+}
+.choice {
+  padding-top: 30px;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+.choice .button1 {
+  font-size: xx-large;
+  background-color: #88c8bc;
+  border-radius: 20px;
+  margin-right: 30px;
+}
+.choice .button2 {
+  font-size: xx-large;
+  background-color: #88c8bc;
+  border-radius: 20px;
+}
+
+.container {
+  width: 90%;
+  height: 400px;
+  display: flex;
+  justify-content: center;
+  justify-content: space-around;
+  padding-top: 50px;
+  padding-right: 200px;
+}
+.product1 .img1 {
+  width: 200px;
+  height: 200px;
+  padding-left: 10%;
+  padding-top: 10%;
+}
+.img2 {
+  width: 200px;
+  height: 200px;
+  padding-left: 10%;
+  padding-top: 10%;
+}
+.img3 {
+  width: 200px;
+  height: 200px;
+  padding-left: 10%;
+  padding-top: 10%;
+}
+
+.product1 .information1,
+.information2 {
+  display: flex;
+  justify-content: space-around;
+}
+
+.product2 .information1,
+.information2 {
+  display: flex;
+  justify-content: space-around;
+}
+.product3 .information1,
+.information2 {
+  display: flex;
+  justify-content: space-around;
+}
+.product1 {
+  width: 20%;
+  height: 300px;
+  border: 2px #88c8bc solid;
+}
+.product2 {
+  width: 20%;
+  height: 300px;
+  border: 2px #88c8bc solid;
+}
+
+.product3 {
+  width: 20%;
+  height: 300px;
+  border: 2px #88c8bc solid;
+}
+
+.footer {
+  width: 100%;
+  height: 140px;
```

Comment:
**Remove** `height: 140px;` because footer **Automatically** increase the height When you add another element in it . and you can **Add** for example icons for Instagram , Facebook ,...etc. and you can **Add** `E-commerce` instead of `Logo`  as an anchor that when you click on it , it will redirect the view to the Home Page 👌✨